util/walletAddress.ts

```diff
@@ -0,0 +1,18 @@
+import { createRailgunWallet } from "@railgun-community/quickstart";
+import { entropyToMnemonic, hexValue, randomBytes } from "ethers/lib/utils.js";
+
+/**
+ * @notice Tool to generate a random railgun address.
+ * @returns Railgun address (0zk...123).
+ */
+export const randomWalletAddress = async () => {
```

Comment:
What was this for? Seems it's unused anyway, so should we delete it?

Reply:
sounds good

Reply:
removed, thanks!